lib/grape-rabl/formatter.rb

```diff
@@ -4,84 +4,96 @@ module Grape
   module Formatter
     module Rabl
       class << self
-        attr_reader :env
-        attr_reader :endpoint
-
         def call(object, env)
-          @env = env
-          @endpoint = env['api.endpoint']
-
-          if rablable?
-            rabl do |template|
-              engine = tilt_template(template)
-              output = engine.render endpoint, locals
-              if layout_template
-                layout_template.render(endpoint) { output }
-              else
-                output
-              end
-            end
-          else
-            Grape::Formatter::Json.call object, env
-          end
+          GrapeRabl::Formatter.new(object, env).render
         end
+      end
+    end
+  end
+end
+
+module GrapeRabl
```

Comment:
I think ```GrapeRabl``` is the module name being used in other files, so I continued to use the same name here for classes that are ```GrapeRabl``` specific. As for ```Grape::Formatter::Rabl```, it is more a Grape convention for formatters. I will move the Grape::Formatters::Rabl to ```lib/grape``` directory instead. Thanks!